docs/pages/desktop-access/active-directory.mdx

```diff
@@ -0,0 +1,216 @@
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
+  In this guide, we'll install Active Directory on this server in order
+  to support passwordless logins with Teleport to the Windows desktops
+  in the Active Directory domain.
+- A Linux host where you will run the Teleport Desktop Service.
+  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
+  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
+  Access into an existing Teleport instance running other services, see the
+  [Manual Setup](active-directory-manual.mdx) guide for Desktop Access.</ScopedBlock>
+- An Active Directory domain, configured for LDAPS (Teleport requires an
+  encrypted LDAP connection). Typically this means installing
+  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+
+## Step 1/2. Install Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/2. Run the discovery wizard
+
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
+`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
+`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
+and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+
+If you already have Active Directory installed, skip to the next step. Otherwise,
+copy and paste the first command provided into a Windows PowerShell window. If
+you aren't already running AD Certificate services, copy and paste the second
+command after the first one completes and the server restarts:
+
+![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+
+Once the server is restarted from one or both command executions, click **NEXT**.
+
+Copy and paste the provided command into a Windows PowerShell window to download
+and run the 'configure Active Directory' script:
+
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+
+Click **NEXT**.
+
+The PowerShell script will output a Teleport configuration block. Copy this
+block to a temporary location. Click **Next**.
+
+On the Linux host you installed Teleport to run as the Desktop Access connector,
+edit `/etc/teleport.yaml` and paste the configuration provided by the output of
+the previous step. Review and edit the addition as necessarily for your environment:
+
+```yaml
+version: v3
+teleport:
+  auth_token: 2239...c5b21
+  proxy_server: teleport.example.com:443
+
+auth_service:
+  enabled: no
+ssh_service:
+  enabled: no
+proxy_service:
+  enabled: no
+
+windows_desktop_service:
+  enabled: yes
+  ldap:
+    addr:     '10.10.1.50:636'
+    domain:   'windows.teleport.example.com'
+    username: 'WIN\svc-teleport'
+    server_name: 'windows-server-hostname'
+    insecure_skip_verify: false
+    ldap_ca_cert: |
```

Comment:
Just to double-check, does the automation we mentioned earlier set up TLS on the LDAP server and populate this field automatically?

Reply:
What automation are you referring to? If you're talking about the windows installer exe, note that that bin is run on standalone windows machines for non-ad access, separate from this doc.

This doc is just what _was_ the former getting started guide, [renamed](https://github.com/gravitational/teleport/pull/19311/files#r1046457925) to accommodate the convention of non-AD being the default instructions.

Reply:
If you're talking about the Wizard in the Teleport UI, then yes; the config generated should include the AD CA.... 90% confident.

Reply:
👍 I suppose I'm really asking, do we need to include instructions for how to supply the value of this field?

Reply:
It might not hurt, but let's track this separately since it's not related to the non-AD use case.

Reply:
Since the text above says:
>paste the configuration provided by the output of
>the previous step. Review and edit the addition as necessarily for your environment

We should include all the content provided in the paste, including this field.